src/script/cryptography/CryptographyService.ts

```diff
@@ -17,23 +17,20 @@
  *
  */
 
-import {getLogger} from 'Util/Logger';
+import {APIClient} from '@wireapp/api-client';
+import {ClientPreKey, PreKey} from '@wireapp/api-client/dist/auth';
+import {UserClients} from '@wireapp/api-client/dist/conversation';
+import {UserPreKeyBundleMap} from '@wireapp/api-client/dist/user';
 
 export class CryptographyService {
-  static get CONFIG() {
-    return {
-      URL_CLIENTS: '/clients',
-      URL_USERS: '/users',
-    };
-  }
+  private readonly apiClient: APIClient;
 
   /**
    * Construct a new Cryptography Service.
-   * @param {BackendClient} backendClient - Client for the API calls
+   * @param {APIClient} apiClient - Client for the API calls
```

Comment:
```suggestion
   * @param apiClient Client for the API calls
```

Reply:
Let's remove the whole comment block since the types are self-explaining.